renderer.cpp

```diff
@@ -740,3 +727,10 @@ void Renderer::makeScreenshot(const FilePath& path) {
   for (int i : Range(bmpSize))
     output << bitmap[i];
 }
+
+int Renderer::getWidth() const {
```

Comment:
There is already a getSize() method. Btw, at some point we need to test if all your changes work with the Zoom UI setting, which enlarges everything 2x for high dpi monitors.